### DISCUSSION TITLE
blog/deploying-ruby-on-rails-on-render-with-databse-redis-sidekiq-cron

### DISCUSSION THREAD

**Discussion by giscus** [harrison-broadbent/railsnotes-comments#4] [RailsNotes Comments]
# blog/deploying-ruby-on-rails-on-render-with-databse-redis-sidekiq-cron

A full walkthrough guide for deploying a Ruby on Rails app to Render with a database, Redis, Sidekiq, and Cron jobs...

https://railsnotes.xyz/blog/deploying-ruby-on-rails-on-render-with-databse-redis-sidekiq-cron

**Comment by GenaroLegaspiJr** [comment]
How do you access the sidekiq dashboard on Render?

**Reply by harrison-broadbent** [reply]
If you're mounting the Sidekiq dashboard to a web url (`mount Sidekiq::Web => '/sidekiq' in your routes`), then you should just be able to access it via the `/sidekiq` url as usual.
